packages/editor/src/components/post-saved-state/index.js

```diff
@@ -48,10 +47,8 @@ export default function PostSavedState( {
 		isDirty,
 		isNew,
 		isPending,
-		isPublished,
 		isSaveable,
 		isSaving,
-		isScheduled,
```

Comment:
Also we removed these two return values but forgot to remove the keys within the useSelect callback.